docs/book/src/user/quick-start.md

```diff
@@ -517,7 +517,7 @@ The output of `clusterctl init` is similar to this:
 
 ```bash
 Fetching providers
-Installing cert-manager Version="v1.8.2"
```

Comment:
And yet we always do this :) (same for Kubernetes version)

I'll merge given that it's also fine to update it.

Reply:
It's weird here because the cluster api version and the cert manager version are out of sync so it could be confusing.

Reply:
Not sure if they are actually out of sync. This doc is written for main not for release-1.2. If we want to keep the version up-to-date we should do it know. Otherwise I would just not bump them going forward (as I don't want to do it *at* the time of the release)

Reply:
IMO let's merge as is and I'll do a cleanup in a follow-up PR, we can also make a point of intentionally updating (or not) these versions going forward, but probably the worst thing to do is to update them haphazardly, if something went wrong with init  a person might spend time wondering why their version output is very different than that given in the logs (both older and newer in this case if installing 1.2)

Reply:
That's fine, but our current idea is that the documentation on main matches the implementation on main. 

I would really prefer not intentionally diverging between doc and implementation and then having a list of things we *have* to update in the documentation at the exact time of the release (except there's no way around it)

Reply:
>That's fine, but our current idea is that the documentation on main matches the implementation on main.

So then we should update the provider versions in the logs as well as Cert Manager, or at least leave them both out of date or in line with each other. Here we have cert manager v1.9.1 and CAPI v1.0.0 which is a pretty big drift. I'd be fine with either updating or reducing the version here, but only updating cert manager doesn't seem like the right change IMO

Reply:
Agree, that makes sense!

I'll merge and let you follow-up. Thx!

Reply:
I have a slight tendency towards just having some old CAPI / cert-manager version so folks can just obviously see it's not 100% identical to what they have. 

I think it's not super important that we document the exact output here and keep it always up-to-date. It's after all a relatively straightforward log and in my opinion not worth the effort of always keeping it sync

Reply:
100 percent agreed - it's just the version drift that can be confusing.